mappings/net/minecraft/block/Block.mapping

```diff
@@ -336,7 +335,7 @@ CLASS none/aky net/minecraft/block/Block
 	METHOD j getRawIdFromBlockState (Lnone/asm;)I
 		ARG 0 state
 	METHOD k setUnbreakable ()Lnone/aky;
-	METHOD k canPlaceThingsOn (Lnone/asm;)Z
+	METHOD k hasSolidTopSurface (Lnone/asm;)Z
```

Comment:
Oooohh.. I get it now! I think we should make sure this has a similar name to the other function that's used to see if levers/torches/buttons can be placed on the side / bottom of them as well.